Modules/Core/Common/include/itkPolyLineCell.hxx

```diff
@@ -0,0 +1,277 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkPolyLineCell_hxx
+#define itkPolyLineCell_hxx
+
+namespace itk
+{
+/**
+ * Standard CellInterface:
+ */
+template <typename TCellInterface>
```

Comment:
@PranjalSahu as metioned in commit https://github.com/InsightSoftwareConsortium/ITK/pull/3351/commits/a837b20eb398a3dfc023cd3f62c5d1b4fad26ee0, methods need to be documented in header files. Probably this is a side effect of other `CellInterface` classes being documented in the wrong place as well.

Also, virtually all child methods seem to have the exact same documentation (have checked only a few methods), so if I'm not mistaken, if a overridden method is not documented, the parent class documentation is used for the child class. The latter should be verified, but if it is the case (should be the case for e.g. `PrintSelf` methods, although there are some extraneous cases #3401), and if the reimplementation does not require any custom documentation, then all method documentation should be removed. That would ease maintenance as well.

You may want to check and fix this. Thanks.

Reply:
ok I will move them to header files.